src/theVoice/index.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components'
+import { space, font } from '_utils/branding'
+
+import TheVoice from './TheVoice'
+
+const StyledTheVoice = styled(TheVoice)`
+  &.rm-small {
+    font-size: ${font.l.size};
+    margin: ${space.xl} 0;
+  }
+  
+  &.rm-large {
+    text-align: center;
+    font-size: ${font.xl.size};
```

Comment:
Not the same font-size ? On large screen ?

Reply:
Yep it's part of the specs, it get smaller for small viewports.
We might need to renormalize all our font sizes, none of Dawid's spec values match what we have in our branding config. For now I just use a smaller font.